src/Account.ts

```diff
@@ -364,5 +364,35 @@ export class Account {
     return this.reachStdLib.formatAddress(contractAddress);
   }
 
-  // TODO: ADD listeners for events
+  /**
+   * Subscribes to all vault events and calls the provided callbacks when an
+   * event is fired.
+   */
+  async subscribeToEvents(params: {
```

Comment:
@tomxbacked apologies, I should have caught this. Please use `typedoc` to annotate the funciton, so that it builds correctly for the SDK docs.